stormbench/streaming/dependency-reduced-pom.xml

```diff
@@ -0,0 +1,85 @@
+<?xml version="1.0" encoding="UTF-8"?>

```

Comment:
Why do you change the `pom.xml` for stormbench? do they have dependency? can we do it in another PR if you feel some bugs in stormbench?

Reply:
it's a typo.